src/cron/syncSubscriptionWithCIO.ts

```diff
@@ -44,6 +46,7 @@ const cron: Cron = {
         break;
       }
     }
+    logger.info(`synced subscriptions for ${syncedSubscriptions} users`);
```

Comment:
You don't have to do it retro but better to pass as JSON object eg. `logger.info({ count: syncedSubscriptions }, 'synced subscriptions')`

Reply:
True, that's better. See #2030